### PR TITLE
ci: auto fetch latest sdk version

### PR DESCRIPTION
**Pull request by HeZean**

### CODE REVIEW

**Comment by HeZean** [comment]
PTAL @FengShang, does that match your requirement?

**Review by FengShang — COMMENTED** [review]
image

This line should also be replaced.

**Comment by FengShang** [comment]
> PTAL @FengShang, does that match your requirement?

Yes, Great!

File: .github/sdk_version_updater.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    return resp['crate']['max_stable_version']
+
+
+def update_java_docs():
+    JAVA_DOC_PATH = 'docs/sdk/Server-Side SDKs/java-sdk.md'
+    with open(JAVA_DOC_PATH, 'r', encoding='utf-8') as f:
+        content = f.read()
+        content = re.sub('<artifactId>server-sdk-java</artifactId>\n    <version>.*</version>', f'<artifactId>server-sdk-java</artifactId>\n    <version>{sdk_versions["java"]}</version>', content)
+        content = re.sub("implementation 'com\\.featureprobe:server-sdk-java:.*'", f"implementation 'com.featureprobe:server-sdk-java:{sdk_versions['java']}'", content)
+        content = re.sub("./target/server-sdk-java-.*\\.jar", f"./target/server-sdk-java-{sdk_versions['java']}.jar", content)
```
Comment (HeZean) [inline .github/sdk_version_updater.py:53]: @FengShang will be updated by this line of code

**Pull request merged**